### ISSUE TITLE
Dislike Counter clearly in the Wrong Place

### ISSUE THREAD

**Issue by Obscure2020** [revanced/revanced-patches#1140]
### Type

Cosmetic

### Bug description

The dislike counter in ReVanced is currently being shown in CLEARLY the wrong position. The dislike counter should be shown on the Dislike button, not appended to the like count on the Like button!

### Steps to reproduce

Step 1: Compile YouTube ReVanced with APK v17.45.36, cli v2.16.1, and patches v.2.121.0, **including the Return YouTube Dislike patch.**
Step 2: Open the app.
Step 3: Open pretty much any video.

### Relevant log output

```shell
N/A (This is a cosmetic issue, not an error issue.)
```

### Screenshots or videos

This is how it currently looks in ReVanced:\
![cropped_ReVanced](https://user-images.githubusercontent.com/69320464/204363167-4dade9c0-1628-4abf-9bd9-0d78e390a898.png)

Here is how it *should* look (how it looks on desktop):\
![desktop_upscaled](https://user-images.githubusercontent.com/69320464/204363437-b172e670-fc98-402c-829d-367a8206739b.png)

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by Ushie** [comment]
This is due to current limitations when it comes to the new design, it is not yet figured out how to add a new text element next to the dislike counter as the new YouTube design does not account to it, hence it'd require much more work.

**Comment by Ushie** [comment]
Needless to say, this is intended for now, we can't expect people to spend more of their free time than necessary just to get the text to display in the current section

**Comment by Obscure2020** [comment]
Yeah, that's what I was expecting. I had a feeling YouTube's new layout would take some time for the devs to get acquainted with. There's no rush on this request. Just wanted to let you guys know that you'll have my sincerest gratitude if you ever get around to remedying this.

**Comment by KraXen72** [comment]
+1, i'd like to see ryd show the dislikes where they belong
